examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -328,7 +352,7 @@ func copyableType(t *types.Type) bool {
 		return false
 	}
 	// TODO: Consider generating functions for other kinds too.
-	if t.Kind != types.Struct {
+	if t.Kind != types.Struct && !(t.Kind == types.Alias && hasPostDeepCopyMethod(t)) {
```

Comment:
This makes alias+hasPostDeepCopyMethod copyable. So the question is why the non-existence of postDeepCopy on an alias makes it uncopyable. I think it shouldn't.